hs_core/templates/pages/genericresource.html

```diff
@@ -800,19 +804,21 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             {#  Resource specific metadata can't be edited for HIS Referenced Time Series #}
                             {% if cm|resource_type != "HIS Referenced Time Series" or not metadata_form %}
                                 {% if  cm|resource_type != "Generic" %}
-                                    <div class="tab-pane" id="resourceSpecificTab">
-                                        {% if not metadata_form %}
+                                    {% if  cm|resource_type|lower != "collection resource" %}
```

Comment:
@zhiyuli I think you should retain "Generic" check in the if condition rather than replace it.